libpod/info.go

```diff
@@ -146,3 +153,21 @@ func readUptime() (string, error) {
 	}
 	return string(f[0]), nil
 }
+
+// GetConmonVersion returns a string representation of the conmon version
+func (r *Runtime) GetConmonVersion() (string, error) {
+	output, err := utils.ExecCmd(r.conmonPath, "--version")
+	if err != nil {
+		return "", nil
```

Comment:
This and the one below should probably be `return "", err` - won't really change anything since we throw the errors away above, but if we ever reuse this it's worth it to return them

Reply:
sure, fixed